### ISSUE TITLE
Confusing key roles in security.yaml > access_control

### ISSUE THREAD

**Issue by TomPradat** [symfony/symfony#31328]
**Description**  
I know this is not a new topic but I find the key roles in the access_control part very disturbing, i spend hours to understand that i can put attributes there too.

I think this is not clear in the documentation.

Maybe we should juste update the documentation to make it clear or change the key name for something else

**Comment by TomPradat** [comment]
This is related to https://github.com/symfony/symfony/issues/21029

**Comment by linaori** [comment]
In particular relates to my comment in https://github.com/symfony/symfony/issues/21029#issuecomment-269291384: _Problem 2 - Unintuitive behavior_

**Comment by chalasr** [comment]
:+1: for renaming

**Comment by wouterj** [comment]
I'm all in for renaming, but I don't think we've yet come up with a great alternative (we're also struggling with this in the documentation).

`attribute` is imho not very describing (and in fact confusing, as `Token#getAttributes()` is not related to this attribute concept, see also e.g. https://github.com/symfony/symfony-docs/issues/4158).

**Comment by mab05k** [comment]
Is this syntax for configuring roles no longer acceptable? (since upgrading from 4.4.x to 5.x.x)
```
    access_control:
        - path: ^/api/instruments.*
          roles:
            - ROLE_PERFORMANCE_READ
            - ROLE_ACCOUNT_READ
            - ROLE_CONFIGURATION_READ
          methods: [ GET ]
```
Replacing the `roles` section with
```
allow_if: "is_granted('ROLE_PERFORMANCE_READ') or is_granted('ROLE_ACCOUNT_READ') or is_granted('ROLE_CONFIGURATION_READ')"
```
seems to fix my issue, but seems less intuitive and is not well documented.

**Comment by linaori** [comment]
@mab05k should be fixed in 5.0.8: #36283
